lib/vsc/utils/run.py

```diff
@@ -899,7 +899,7 @@ def _loop_process_output(self, output):
 
         # qa first and then qa_reg
         nr_qa = len(self.qa)
-        for idx, (question, answers) in enumerate(self.qa.items() + self.qa_reg.items()):
+        for idx, (question, answers) in enumerate(list(self.qa.items()) + list(self.qa_reg.items())):
```

Comment:
oh boy, why couldn't they implement a add function
also, this is (well, already was) unsorted?

Reply:
Yeah, these should really be sorted (but individually, otherwise entries in `self.qa_reg` could get precedence over `self.qa` items, which may break stuff... Will fix.

Reply:
sorted now (but still separately for `self.qa` and `self.qa_reg`)